src/en/web/design/forms.md

```diff
@@ -39,3 +40,47 @@ Lastly, the wording of the error messages should be explicit.
 
 **Don’t:**  
 ![screenshot of a form displaying irrelevant error messages](../../images/formulaire-ko.png)
+
+## Limit redundant entries
+
+**Target:**  People with cognitive disabilities (especially with memory difficulties).
+
+**When:** During design.
```

Comment:
It implies thinking about how to store informations during the session and/or process, using the correct value for `autocomplete` attribute, auto-populating inputs, etc. SOunds more like a dev thing to me no?

Reply:
design and development